### DISCUSSION TITLE
estou com uma dificuldade para instala o photoshop

### DISCUSSION THREAD

**Discussion by geansouzaribeiro** [community/community#65027] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

======+====+===========+=======================================================
bf181c|ERR |       0B/s|/home/gean/.cache/photoshopCCV19/photoshopCC-V19.1.6-2018x64.tgz

Status Legend:
(ERR):error occurred.

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.
downloading photoshopCC-V19.1.6-2018x64.tgz ...
using aria2c to download photoshopCC-V19.1.6-2018x64.tgz

08/26 08:54:07 [NOTICE] Downloading 1 item(s)

08/26 08:54:08 [ERROR] CUID#7 - Download aborted. URI=https://victor.poshtiban.io/p/gictor/photoshopCC/photoshopCC-V19.1.6-2018x64.tgz
Exception: [AbstractCommand.cc:351] errorCode=19 URI=https://victor.poshtiban.io/p/gictor/photoshopCC/photoshopCC-V19.1.6-2018x64.tgz
  -> [AbstractCommand.cc:792] errorCode=19 CUID#7 - Name resolution for victor.poshtiban.io failed:Misformatted domain name

08/26 08:54:08 [NOTICE] Download GID#bf7fc08a3bf3bc9c not complete: /home/gean/.cache/photoshopCCV19/photoshopCC-V19.1.6-2018x64.tgz

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
bf7fc0|ERR |       0B/s|/home/gean/.cache/photoshopCCV19/photoshopCC-V19.1.6-2018x64.tgz

Status Legend:
(ERR):error occurred.

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.
error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz
➜  photoshopCClinux git:(master)

pode me ajudar?

**Comment by ghostinhershell** [comment]
Hi @geansouzaribeiro, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
